content/en/docs/reference/glossary/container-lifecycle-hooks.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Container Lifecycle Hooks
+id: container-lifecycle-hooks
+date: 2018-10-08
+full_link: /docs/concepts/containers/container-lifecycle-hooks/
+short_description: >
+  The lifecycle hooks expose events in the container management lifecycle and let the user run code when the events occur.
+
+aka: 
+tags:
+- extension
+---
+  The lifecycle hooks expose events in the {{< glossary_tooltip text="Container" term_id="container" >}}container management lifecycle and let the user run code when the events occur.
+
+<!--more--> 
+
+There are two hooks that are exposed to Containers: PostStart which executes immediately after a container is created and PreStop which is blocking and is called immediately before a container is terminated.
```

Comment:
s/There are two hooks that/Two hooks
(tighten up text a bit)

Reply:
Thanks for the review !
I applied your suggestion.